universe/ValueRef.cpp

```diff
@@ -611,6 +611,11 @@ namespace ValueRef {
         } else if (property_name == "CurrentTurn") {
             return CurrentTurn();
 
+        } else if (property_name == "Attack") {
+            if (TemporaryPtr<const Fleet> fleet = boost::dynamic_pointer_cast<const Fleet>(object))
+                return fleet->Damage();
+            if (TemporaryPtr<const Ship> ship = boost::dynamic_pointer_cast<const Ship>(object))
+                return ship->TotalWeaponsDamage();
```

Comment:
Definitely think keep planets out, and if put in somewhere make it current stat not potential stat, then it grows every turn rather than just jumping up rapidly, and if you've got a fleet blockading everywhere it should show zero, not whatever potential it won't be at again.